README.md

```diff
@@ -1,7 +1,89 @@
+
+Unlike other submission management platforms Hypha is self-hosted.
+
+This means that the applications, and all associated information created is available to you and your applicants on your own systems.
+
+## Have a need? Modify Hypha to expand
+Hypha is open source software. This means the license invites you to expand what can do Hypha, create add-on functionality not already available.
+
+All we ask is you contribute the improvements back to the project so others can benefit.
+
+# Technology
+
+Hypha is built with:
+- [Django](https://www.djangoproject.com/), [Wagtail](https://wagtail.io/), and [React](https://reactjs.org/)
+and deployed with:
+- [Heroku](https://github.com/OpenTechFund/hypha/wiki/Deployment:-Heroku), [Docker](https://github.com/OpenTechFund/hypha/wiki/Deployment:-Docker), or [your own server](https://github.com/OpenTechFund/hypha/wiki/Deployment:-Standalone-App).
+
+# Who uses Hypha?
+
+Thank you brave folks who choose to use Hypha <3:
+- [Open Technology Fund](https://www.opentech.fund/)
+- [Reset](https://www.reset.tech/)
+
+# About the Hypha team
+
+Hypha is maintained by:
+- [Sarah Aoun](https://github.com/saoun) @ [Open Technology Fund](https://www.opentech.fund/)
+- [Dan Blah](https://github.com/danblah) @ [Reset](https://www.reset.tech/)
+- [Fredrik Jonsson](https://github.com/frjo) @ [Combonetwork](https://www.combonet.se/)
+
+with thanks to all the people who have contributed code & UX design to Hypha: <3
+- [Torchbox](https://www.torchbox.com/)
+- [Remote Inning](https://www.remoteinning.com/)
+- [Maxwell Pearl](https://maxwellpearl.com/)
+- [Throneless Technology](https://throneless.tech/)
+- [Psycle](https://psycle.com/)
+
+# Support Hypha
+
+Hypha is an open source project made possible thanks to the support of sponsors.
+
+Thank you to the following for their genorosity: <3
+
+- [Open Technology Fund](https://www.opentech.fund)
+- [Reset](https://www.reset.tech)
+
+If you rely on Hypha, please consider becoming a sponsor to help those contributing code. Hypha can be supported by anyone and the money will be distributed directly to contributors.
+
+You can provide general support via [Open Collective](https://opencollective.com/hypha) or [Liberapay](https://liberapay.com/hypha).
+
+You can sponsor specific issues via [Issuehunt](https://issuehunt.io/r/OpenTechFund/hypha). Thanks in advance <3
+
+[![Sponsor with Liberapay](https://liberapay.com/assets/widgets/donate.svg)](https://liberapay.com/hypha)
+
+[![Sponsor with Open Collective](https://opencollective.com/hypha/tiers/backer.svg?avatarHeight=75&width=600)](https://opencollective.com/hypha)
+
+[![Find and sponsor issues](https://issuehunt.io/static/embed/issuehunt-button-v1.svg)](https://issuehunt.io/r/OpenTechFund/hypha)
 
-Submission management software for open calls
 
 This is the repo for the [Hypha site](https://www.hypha.app/).
 
-[Hypha code](https://github.com/OpenTechFund/hypha)
+[Hypha code](https://github.com/HyphaApp/hypha)
```

Comment:
This will be true some time in the future but for now the main repo is OTF:s.

Reply:
Sure. I was wondering about that. I'll change it back now.